### PR TITLE
Updating PropPageDesignerWindowPane.PreProcessMessage to not handle Alt+Down

### PR DESCRIPTION
**Pull request by tannergooding**
This replaces #2061 and resolves VSO Bug 409664.

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @dotnet/project-system, @davkean 

I'm still trying to determine whether or not Dev10 bug 818320 is resolved. If it is, it might be possible to just let the shell handle the codes normally (there may be other built-in accelerators that are being missed currently).

**Comment by tannergooding** [comment]
Dev10 Bug 818320 was resolved 7 years ago (and does not repro in the current build). I've updated the PR to drop the special handling accordingly.

**Comment by tannergooding** [comment]
Nevermind, The Dev10 bug did not regress. The "fix" was this code. So the only "valid" fix here is to extend it as I have done in the PR.

**Comment by jmarolf** [comment]
:shipit:

File: src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerWindowPane.vb [hunk 1]
```diff
@@ -179,14 +179,15 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
...
+                    'We must never handle messages in three cases: if this is a VK_MENU, which means the user just pressed alt
+                    'and did nothing more), if this is a VK_DOWN, which is used by certain controls (such as ComboBox) to expand
+                    ' themselves, Or if shift key is being held down. This is to allow Alt+Shift+letter to go to the menus or do
+                    ' whatever Else they are bound To
 
                     'GetKeyState returns with the the highest of the 16-bit value set if the key is being pressed
                     Dim shiftBeingHeld As Boolean = (NativeMethods.GetKeyState(Keys.ShiftKey) And &H8000) <> 0
 
-                    If KeyCode <> Keys.Menu AndAlso Not shiftBeingHeld Then
+                    If KeyCode <> Keys.Menu AndAlso KeyCode <> Keys.Down AndAlso Not shiftBeingHeld Then
```
Comment (davkean) [inline src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerWindowPane.vb]: I think you want at least Keys.Up as well, as ALT+UP hides the active list.

**Comment by davkean** [comment]
Not really liking their original fix, but am okay with the change based on its existence.

File: src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerWindowPane.vb (hunk 1 above)
Comment (tannergooding) [inline src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerWindowPane.vb]: Fixed. I imagine there are other key codes we are missing out on...

**Comment by tannergooding** [comment]
Going to tag @jasonmalinowski, since he implemented the existing logic here. Maybe he has some idea on a better fix here...

**Comment by jasonmalinowski** [comment]
@tannergooding You presume I understood the logic I was implementing. But this looks reasonable given the original hack.

**Pull request merged**